editor.planx.uk/src/pages/layout/FlowEditorLayout.tsx

```diff
@@ -1,9 +1,13 @@
 import ErrorFallback from "components/ErrorFallback";
+import FlowEditor from "pages/FlowEditor";
 import React, { PropsWithChildren } from "react";
 import { ErrorBoundary } from "react-error-boundary";
 
 const FlowEditorLayout: React.FC<PropsWithChildren> = ({ children }) => (
-  <ErrorBoundary FallbackComponent={ErrorFallback}>{children}</ErrorBoundary>
+  <ErrorBoundary FallbackComponent={ErrorFallback}>
+    <FlowEditor />
```

Comment:
This is the single `FlowEditor` instance, attached to the layout, which persists regardless of route.